### DISCUSSION TITLE
Huggingface model in Mindsdb

### DISCUSSION THREAD

**Discussion by arcontechnologies** [mindsdb/mindsdb#2352] [Get help]
Hi,

Can someone give a pratical example on how to use Huggingface model in Mindsdb ?

Thanks in advance

**Comment by paxcema** [comment]
Hi @arcontechnologies, I think the best option as of now is to wrap the HF model in either MLflow or Ray Serve and use it that way. Here's a [blog post](https://medium.com/p/47a188d6db00) that goes through an example.

Let me know if this works or if you are after something perhaps different :v:.

**Reply by arcontechnologies** [reply]
Thanks, I'll give it a try 👍

**Reply by ZoranPandovski** [reply]
@torrmal started working on Huggingface implementation which is WIP atm. If you want to contribute, you can send him a message on slack :raised_hands:
